Index: src/game_entities/Board.java
===================================================================
diff --git a/src/game_entities/Board.java b/src/game_entities/Board.java
deleted file mode 100644
--- a/src/game_entities/Board.java	(revision 758520cc7fe50cff86d3c17ba3e5c0552e1bd326)
+++ /dev/null	(revision 758520cc7fe50cff86d3c17ba3e5c0552e1bd326)
@@ -1,16 +0,0 @@
-package game_entities;
-
-import java.util.ArrayList;
-
-public class Board implements BoardGraph{
-
-    @Override
-    public ArrayList getEnemyNeigbours(Object o) {
-        return null;
-    }
-
-    @Override
-    public ArrayList getAlliedNeighbours(Object o) {
-        return null;
-    }
-}
Index: src/game_entities/BoardGraph.java
===================================================================
diff --git a/src/game_entities/BoardGraph.java b/src/game_entities/BoardGraph.java
deleted file mode 100644
--- a/src/game_entities/BoardGraph.java	(revision 758520cc7fe50cff86d3c17ba3e5c0552e1bd326)
+++ /dev/null	(revision 758520cc7fe50cff86d3c17ba3e5c0552e1bd326)
@@ -1,17 +0,0 @@
-package game_entities;
-
-import java.util.ArrayList;
-
-public interface BoardGraph<Territory> {
-    /*
-     * This gives the list of territories that are neighbouring the player's territory that aren't owned by the player
-     * (territories that can be attacked)
-     */
-    public ArrayList<Territory> getEnemyNeigbours(Territory territory);
-
-    /*
-     * This is the complete list of allied territories that has a path from the given territory.
-     */
-    public ArrayList<Territory> getAlliedNeighbours(Territory territory);
-
-}
